CONTRIBUTING.md

```diff
@@ -155,7 +155,7 @@ mvn -f portfolio-app/pom.xml clean verify
 
 ```
 set MAVEN_OPTS="-Xmx4g"
```

Comment:
Or this is the Windows section and I'm wrong. But it says "Linux/macOS". :man_shrugging: